src/tools/optimization-options.h

```diff
@@ -123,6 +123,36 @@ struct OptimizationOptions : public ToolOptions {
            [this](Options* o, const std::string& argument) {
              passOptions.shrinkLevel = atoi(argument.c_str());
            })
+      .add("--always-inline-max-function-size",
+           "-aimfs",
+           "Max size of functions that are always inlined (default 2, which "
+           "is safe for use with -Os builds)",
```

Comment:
I wonder if perhaps it's safer to not mention the default values here, since they will likely get out of date.

Reply:
Sure, but people wanting to tweak these need to know what to change them from? If I have no idea what is a good value for these parameters, that will be hard to use.. so the current values need to be visible somewhere. I could actually pick up the current defaults from `InliningOptions` if I have access to that header..

Reply:
Another option might be say "look at pass.h"? I think maybe someone doing low-level inlining heuristic tweaking may be ok with reading in the source.

But maybe I'm overthinking it. I'm not strongly opposed to hardcoding them here if you feel that's best.

Reply:
Check the latest commit, I've un-hardcoded them.

Reply:
Oh cool, I see. Nice, lgtm!